scala/private/phases/phase_scala_provider.bzl

```diff
@@ -0,0 +1,45 @@
+#
+# PHASE: scala provider
+#
+# DOCUMENT THIS
+#
+load(
+    "@io_bazel_rules_scala//scala:providers.bzl",
+    "create_scala_provider",
+)
+
+def phase_library_scala_provider(ctx, p):
+    args = struct(
+        rjars = depset(
+            transitive = [p.compile.rjars, p.collect_exports_jars.transitive_runtime_jars],
+        ),
+        compile_jars = depset(
+            p.compile.ijars,
+            transitive = [p.collect_exports_jars.compile_jars],
+        ),
+    )
+    return phase_common_scala_provider(ctx, p, args)
+
+def phase_common_scala_provider(ctx, p, _args = struct()):
+    return _phase_scala_provider(
+        ctx,
+        p,
+        _args.rjars if hasattr(_args, "rjars") else p.compile.rjars,
+        _args.compile_jars if hasattr(_args, "compile_jars") else depset(p.compile.ijars),
+    )
+
+def _phase_scala_provider(
+        ctx,
+        p,
+        rjars,
+        compile_jars):
+    return create_scala_provider(
+        class_jar = p.compile.class_jar,
+        compile_jars = compile_jars,
+        deploy_jar = ctx.outputs.deploy_jar,
+        full_jars = p.compile.full_jars,
+        ijar = p.compile.class_jar,  # we aren't using ijar here
```

Comment:
this looks like a bug, no? Doesn't the [lib](https://github.com/bazelbuild/rules_scala/pull/865/files#diff-a7feab8f41e78c82c1e78f85a9fc48eeL118) impl pass ijars?

Reply:
It does look like a bug. Fixed.

Reply:
Fixed.